containers/signing-service/Dockerfile.j2

```diff
@@ -0,0 +1,15 @@
+FROM debian:{{ debian_version }}
+
+COPY testing.pref                       /etc/apt/preferences.d/
+COPY install-dependencies.sh            /tmp/
+RUN  /tmp/install-dependencies.sh
+
+COPY signing-service/requirements.lock  /srv/http/signing_service/
+COPY install-signing-service.sh         /tmp/
+RUN  /tmp/install-signing-service.sh
+
+USER signingserviceuser
+COPY signing-service                    /srv/http/
```

Comment:
Please add a trailing `/` so that it's apparent that `signing_service/` is a directory.

Reply:
Done.